dev-docs/bidders/bidstack.md

```diff
@@ -29,4 +29,6 @@ The Bidstack Bidding adapter requires setup before beginning. Please contact us
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description  | Example   | Type     |
 |---------------|----------|--------------|-----------|----------|
-| `publisherId` | required | Publisher ID | `'be224bf2-fd3f-4afb-b6fc-4a97718be2f5'` | `string` |
\ No newline at end of file
+| `publisherId` | required | Publisher ID | `'be224bf2-fd3f-4afb-b6fc-4a97718be2f5'` | `string` |
+| `placementId` | optional | Placement ID | `'some_placement_id'` | `string` |
+| `consent` | optional | User consent | `true` | `boolean` |
```

Comment:
User consent for what exactly? Not sure I like the precedence this sets -- bid adapters should be reading the TCF and USP strings -- not bidder-specific consent.

Reply:
Hi @bretg, thank you for the review. 
This attribute indicates that a user consents to a publisher's EULA. It is meant to be used for internal purposes only, adapter only passes this attribute to the Bidstack. TCF consent is expected to be passed separately. If it is advisable - this attribute can be moved to the bid request root.

Reply:
Ok, can you update the description to describe that this is EULA consent please?

Reply:
@bretg done